platform_versioned_docs/version-23.2.0/enterprise/advanced-topics/manual-aws-batch-setup.mdx

```diff
@@ -158,25 +158,27 @@ Create a launch template to configure the EC2 instances deployed by Batch jobs:
 
     #cloud-config
     write_files:
-      - path: /root/tower-forge.sh
+      - path: /root/seqera-ce.sh
         permissions: 0744
         owner: root
         content: |
           #!/usr/bin/env bash
           exec > >(tee /var/log/tower-forge.log|logger -t TowerForge -s 2>/dev/console) 2>&1
           ##
           yum install -q -y jq sed wget unzip nvme-cli lvm2
-          wget -q https://s3.amazonaws.com/amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
+          #Uncomment either the AMD64 or ARM64 line below to install the correct version of Amazon CloudWatch Agent for your CPU architecture
+          #wget -q https://amazoncloudwatch-agent.s3.amazonaws.com/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
+          #wget -q https://amazoncloudwatch-agent.s3.amazonaws.com/amazon_linux/arm64/latest/amazon-cloudwatch-agent.rpm
           rpm -U ./amazon-cloudwatch-agent.rpm
           rm -f ./amazon-cloudwatch-agent.rpm
-          curl -s https://nf-xpack.seqera.io/amazon-cloudwatch-agent/config-v0.3.json \
-            | sed 's/$FORGE_ID/<your prefix>/g' \
+          curl -s https://github.com/seqeralabs/nf-tower-aws/tree/master/cloudwatch-config/agent-config.json \
+            | sed 's/seqera-prefix/<your prefix>/g' \
```

Comment:
1. this command is broken
2. it must be hosted on `nf-xpack.seqera.io` 
3. being a "static" config there's no need to use the `sed` command, the prefix can stay in the config file